install/advanced/core/index.rst

```diff
@@ -846,6 +846,22 @@ Serving {“geonode”, “geoserver”} via NGINX
   cheaper-busyness-backlog-alert = 16  ; Spawn emergency workers if more than this many requests are waiting in the queue
   cheaper-busyness-backlog-step = 2    ; How many emergency workers to create if there are too many requests in the queue
 
+Note
+...........................
+
+Since this is a local environment, Geoserver and Geonode should be able to reach the default MEDIA_ROOT. We need to change the FILE_UPLOAD_DIRECTORY_PERMISSIONS and FILE_UPLOAD_PERMISSIONS in settings.py to make the folder accessible to both.
```

Comment:
HI @Raji-Ahmad 
I would keep it more hipotetical, since everyone can have a different environment. 
What do you think about something like this?

> Geoserver and Geonode should be able to reach the MEDIA_ROOT. If needed, is possible to change the FILE_UPLOAD_DIRECTORY_PERMISSIONS and FILE_UPLOAD_PERMISSIONS in settings.py to make the folder accessible to both.

Reply:
Yes, that's better. Only change permission when necessary.

Reply:
Can you please update the pr? So I can approve and merge it to master